scripts/build_nim.sh

```diff
@@ -127,138 +146,17 @@ build_nim() {
 	# working directory
 	pushd "$NIM_DIR"
 
-	if grep -q skipIntegrityCheck koch.nim; then
-		# Run Nim buildchain
-		. ci/funs.sh
-		NIMCORES=1 nimBuildCsourcesIfNeeded $UCPU
-		bin/nim c --noNimblePath --skipUserCfg --skipParentCfg --warnings:off --hints:off koch
-		./koch --skipIntegrityCheck boot -d:release --skipUserCfg --skipParentCfg --warnings:off --hints:off
-		if [[ "${QUICK_AND_DIRTY_COMPILER}" == "0" ]]; then
-			# We want tools
-			./koch tools -d:release --skipUserCfg --skipParentCfg --warnings:off --hints:off
-		elif [[ "${QUICK_AND_DIRTY_NIMBLE}" != "0" ]]; then
-			# We just want nimble
-			./koch nimble -d:release --skipUserCfg --skipParentCfg --warnings:off --hints:off
-		fi
-	else
```

Comment:
Personally fine with removing this, but it affects building older Nim version. @arnetheduck @tersec ?

Reply:
As far as I'm concerned, nothing before Nim 1.6 exists anymore.

Reply:
does koch ignore unknown flags? if it does that, it's fine - otherwise, we can reduce this section to pass the flag or not depending on whether it's supported or not - seems like a cheap price to pay for the flexibility.